src/main/java/com/minecolonies/achievements/ModAchievements.java

```diff
@@ -0,0 +1,126 @@
+    /**
+     * Max out a builder
+     */
+    public static Achievement achUpgradeBuilderMax = new AchUpgradeBuilderMax("upgrade.builder.max", "upgrade.builder.max", 2, 1)
+            .registerStat();
+
+    /**
+     * Upgrade a builder to lv 1
+     */
+    public static Achievement achBuildingColonist   = new AchBuildingColonist("upgrade.colonist.first", "upgrade.colonist.first", 0, 2)
+            .registerStat();
+    /**
+     * Max out a builder
+     */
+    public static Achievement achUpgradeColonistMax = new AchUpgradeColonistMax("upgrade.colonist.max", "upgrade.colonist.max", 2, 2)
+            .registerStat();
+
+    /**
+     * Upgrade a lumberjack to lv 1
+     */
+    public static Achievement achBuildingLumberjack   = new AchBuildingLumberjack("upgrade.lumberjack.first", "upgrade.lumberjack.first", 0, 3)
+            .registerStat();
+    /**
+     * Max out a lumberjack
+     */
+    public static Achievement achUpgradeLumberjackMax = new AchUpgradeLumberjackMax("upgrade.lumberjack.max", "upgrade.lumberjack.max", 2, 3)
+            .registerStat();
+
+    /**
+     * Upgrade a miner to lv 1
+     */
+    public static Achievement achBuildingMiner   = new AchBuildingMiner("upgrade.miner.first", "upgrade.miner.first", 0, 4)
+            .registerStat();
+    /**
+     * Max out a miner
+     */
+    public static Achievement achUpgradeMinerMax = new AchUpgradeMinerMax("upgrade.miner.max", "upgrade.miner.max", 2, 4)
+            .registerStat();
+
+    /**
+     * Upgrade a fisher to lv 1
+     */
+    public static Achievement achBuildingFisher   = new AchBuildingFisher("upgrade.fisher.first", "upgrade.fisher.first", 0, 5)
+            .registerStat();
+    /**
+     * Max out a fisher
+     */
+    public static Achievement achUpgradeFisherMax = new AchUpgradeFisherMax("upgrade.fisher.max", "upgrade.fisher.max", 2, 5)
+            .registerStat();
+
+    // Sizes
+    /**
+     * Reach {@link AchSizeSettlement#SIZE} citizens
+     */
+    public static Achievement achSizeSettlement = new AchSizeSettlement("size.pioneer", "size.settlement", 2, -2)
+            .registerStat();
+    /**
+     * Reach {@link AchSizeTown#SIZE} citizens
+     */
+    public static Achievement achSizeTown       = new AchSizeTown("size.town", "size.town", 4, -2).registerStat();
+    /**
+     * Reach {@link AchSizeCity#SIZE} citizens
+     */
+    public static Achievement achsizeCity       = new AchSizeCity("size.city", "size.city", 6, -2).registerStat();
+
+    // Achievement pages#+
+    /**
+     * The MineColonies achievement page
+     */
+    public static final AchievementPage pageMineColonies = new AchievementPageMineColonies(achGetSupply, achWandOfbuilding, achBuildingTownhall, achBuildingBuilder, achBuildingColonist, achBuildingLumberjack, achBuildingMiner, achBuildingFisher, achSizeSettlement, achSizeTown, achsizeCity, achUpgradeColonistMax, achUpgradeBuilderMax, achUpgradeLumberjackMax, achUpgradeMinerMax, achUpgradeFisherMax);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Split this 402 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00103)